cmd/aries-js-worker/main.go

```diff
@@ -66,6 +88,29 @@ func main() {
 	select {}
 }
 
+// initAries creates aries agent instance
+// TODO currently running on default opts, should be able pass custom framework opts
+func getAllCommands() []cmdctrl.Handler {
+	msgHandler := msghandler.NewRegistrar()
+
+	a, err := aries.New(aries.WithMessageServiceProvider(msgHandler))
+	if err != nil {
+		panic(err)
+	}
+
+	ctx, err := a.Context()
+	if err != nil {
+		panic(err)
```

Comment:
what is the behaviour of panic in a WASM?

Reply:
@sudeshrshetty  you need to return error as alert or error msg in GUI

Reply:
specifically, inside the web worker's context.

Reply:
yes - the WASM should message the worker to do actions (rather than do it directly).

Reply:
doing console.error() for now